app/decorators/reviewable.rb

```diff
@@ -5,6 +5,15 @@ def class
     __getobj__.class
   end
 
+  def attributes=(attributes)
```

Comment:
No - reviewing is only relevant to being made public if it's the first time it's reviewed. However, the "review" checkbox shouldn't show up on the create form.